applications/FluidDynamicsApplication/custom_strategies/convergence_criteria/vel_pr_criteria.h

```diff
@@ -141,7 +141,7 @@ class VelPrCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpace >
                         DofValue = itDof->GetSolutionStepValue(0);
                         DofIncr = Dx[DofId];
 
-                        KeyType CurrVar = itDof->GetVariable().Key();
+                        const auto& CurrVar = itDof->GetVariable();
                         if ((CurrVar == VELOCITY_X) || (CurrVar == VELOCITY_Y) || (CurrVar == VELOCITY_Z))
```

Comment:
@roigcarlo so this check was never `True`?
I mean, how can comparing `KeyType` against `Variable` even compile? Or aren't those the types?

Reply:
Actually I think it was working before. Taking a closer look, it seems that variable_data has the size_t() operator redefined to it return the key also when implicitly converted to a size_t for the comparion with the key (Which was KeyType).

Anyway, probably is less confusing this way.

Reply:
:+1: I found it less confusing in this way.